lib/OpenQA/Scheduler.pm

```diff
@@ -269,10 +269,6 @@ sub job_create {
     my ($settings, $no_notify) = @_;
     my %settings = %$settings;
 
-    if (my $error = OpenQA::Variables->new()->check(%settings)) {
```

Comment:
I can't find any other usage of Variables::check except this one. Why not remove the Variables.pm completely?